src/sentry/models/environment.py

```diff
@@ -46,7 +60,20 @@ def get_or_create(cls, project, name):
             env = cls.objects.get_or_create(
                 project_id=project.id,
                 name=name,
+                defaults={'organization_id': project.organization_id}
             )[0]
+            if env.organization_id is None:
+                cls.objects.filter(id=env.id).update(
```

Comment:
you could ``env.update(organization_id=project.organization_id)`` which will also setattr on itself